scripts/cron.lua

```diff
@@ -0,0 +1,41 @@
+function compute(process, message, opts)
+    -- Early return when no body is provided
+    -- This handles the initial invocation during process setup
+    if not message.body or not message.body.body then
+        return process
+    end
+    
+    -- Initialize crons table if it doesn't exist
+    process.crons = process.crons or {}
+    
+    local command = message.body.body.path
+    ao.event("debug_cron", { "compute command", command })
+    
+    if command == "put" then
```

Comment:
it may be worth considering to create a function for each command to isolate functionality, I know there is not a lot of moving parts, but you may want to leverage `pcall` to wrap your function code to handle any errors.